styles/_variables.scss

```diff
@@ -23,6 +23,7 @@ $line-height-base: 1.7 !default;
 
 // etc base
 $border-radius-base: 3px;
+$border-radius-base2x: 6px;
```

Comment:
Let's just use `$border-radius-base` at the moment. All components should have uniform `border-radius`. I-increase na lang natin later